be1-go/message/answer/error.go

```diff
@@ -30,17 +30,64 @@ func NewErrorf(code int, format string, values ...interface{}) *Error {
 }
 
 // NewInvalidActionError returns an error with the code -1 for an invalid action.
-func NewInvalidActionError(action string) *Error {
-	return &Error{
-		Code:        -1,
-		Description: fmt.Sprintf("invalid action: %s", action),
-	}
+func NewInvalidActionError(action string, values ...interface{}) *Error {
```

Comment:
this is to stay consistent with the `Sprintf` function. 

```suggestion
func NewInvalidActionErrorf(format string, a ...interface{}) *Error {
```